packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -458,14 +504,230 @@ describe('TokenDetectionController', () => {
 
           await controller.start();
 
-          expect(mockAddDetectedTokens).not.toHaveBeenCalledWith(
+          expect(callActionSpy).not.toHaveBeenCalledWith(
             'TokensController:addDetectedTokens',
           );
         },
       );
     });
   });
 
+  describe('AccountsController:selectedAccountChange', () => {
+    let clock: sinon.SinonFakeTimers;
+    beforeEach(() => {
+      clock = sinon.useFakeTimers();
+    });
+
+    afterEach(() => {
+      clock.restore();
+    });
+
+    describe('when "disabled" is "false"', () => {
```

Comment:
Nit: `false` is a boolean, not a string:

``` suggestion
    describe('when "disabled" is false', () => {

Reply:
Fixed throughout test file: https://github.com/MetaMask/core/pull/3867/commits/ad3593df4a9fdb8c6b55380f6caeb6393adb107a